jetson/percep/pcl.cpp

```diff
@@ -186,177 +186,81 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
 
 }
 
+/* --- Get Angle --- */
+//This function finds the angle off the x axis the
+//line that passes through both these points is
+//Direction of 0 is left and 1 is right
+double PCL::getAngle(int buffer, int direction, std::vector<std::vector<int>> &interest_points,
```

Comment:
Get what angle? The comment addresses this in part but, on top of improving the comment to full answer this, the function name should also be fairly descriptive.

Reply:
fixed